### ISSUE TITLE
When using Firefox, NVDA sometimes doesn't read the "Search" edit field

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1531]
**Reported by elliott94 on 2011-05-28 15:09**
When in Firefox and CTRL-K is pressed to take you to the Search edit field, I've noticed that since upgrading to Firefox 4, NVDA sometimes doesn't read its contents. I'm not sure whether this is a Firefox-spacific bug, but thought that it would be useful to inform you either way. The section of the log below unfortunately doesn't give much info, but should give you a good idea of what's going on.

Input: kb(desktop):control+k
IO - speech.speak (16:04:28):
Speaking [Toolbar  tool bar'](u'Navigation)
IO - speech.speak (16:04:28):
Speaking [using Google  combo box'](u'Search)
IO - speech.speak (16:04:28):
Speaking [ has auto complete'](u'edit)
IO - speech.speak (16:04:28):
Speaking [u'blank']
IO - inputCore.InputManager.executeGesture (16:04:29):
Input: kb(desktop):shift+t
IO - inputCore.InputManager.executeGesture (16:04:29):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:30):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:30):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:30):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:30):
Input: kb(desktop):f
IO - inputCore.InputManager.executeGesture (16:04:30):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:30):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:31):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:31):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:31):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:31):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:31):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:31):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:31):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:31):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:32):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:32):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:32):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:32):
Input: kb(desktop):f
IO - inputCore.InputManager.executeGesture (16:04:33):
Input: kb(desktop):f
IO - inputCore.InputManager.executeGesture (16:04:33):
Input: kb(desktop):f
IO - inputCore.InputManager.executeGesture (16:04:33):
Input: kb(desktop):f
IO - inputCore.InputManager.executeGesture (16:04:33):
Input: kb(desktop):f
IO - inputCore.InputManager.executeGesture (16:04:33):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:33):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:34):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:34):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:34):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:34):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:34):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:34):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:35):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:35):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:35):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:35):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:35):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:35):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:36):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:36):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:36):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:36):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:36):
Input: kb(desktop):backspace
IO - inputCore.InputManager.executeGesture (16:04:36):
Input: kb(desktop):backspace

**Comment by ehollig** [comment]
I am not able to replicate this using Firefox 54.0.1. Commenter states it was not reproducible using Firefox 34.0.5 either. Suggest a close.

**Comment by derekriemer** [comment]
closing as worksforme
